shell/src/main/java/org/apache/accumulo/shell/Shell.java

```diff
@@ -472,27 +472,13 @@ public ClassLoader getClassLoader(final CommandLine cl, final Shell shellState)
   }
 
   private static String getTableContextFromProps(Map<String,String> props) {
-    String tableContext = null;
     for (Entry<String,String> entry : props.entrySet()) {
-      // look for either the old property or the new one, but
-      // if the new one is set, stop looking and let it take precedence
       if (entry.getKey().equals(Property.TABLE_CLASSLOADER_CONTEXT.getKey())
           && entry.getValue() != null && !entry.getValue().isEmpty()) {
         return entry.getValue();
```

Comment:
Since we're only looking for a single property now, I don't think this needs to be in a loop. We should be able to just directly call `props.get(Property.TABLE_CLASSLOADER_CONTEXT.getKey())` and return it.